src/emscripten-optimizer/simple_ast.h

```diff
@@ -1190,10 +1182,10 @@ struct JSPrinter {
       ensure(1);                  // we temporarily append a 0
       char* curr = buffer + last; // ensure might invalidate
       buffer[used] = 0;
-      if (strstr(curr, "infinity")) {
+      if (strstr(curr, "Infinity")) {
         return;
       }
-      if (strstr(curr, "nan")) {
+      if (strstr(curr, "NaN")) {
         return;
```

Comment:
these changes (in this file) may be good, but can you split them out to separate PRs?

Keeping each PR focused on one little thing is much better. It helps bisection in the future, and it helps code review in the present.

Reply:
Are mean separate all stuff relate to `nan` / `infinity`?

Reply:
Yes.